_posts/2015-07-20-third-party-integration.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "Challenges of integrating third-parties"
+date: "2015-07-20 07:20:17"
+icon: cloud
+tags: integration third-party API webservice
+categories: tech-stories
+authors: ["Ulf B."]
+---
+
+Third-party applications are an important part of the service that ePages offers.
+As our applications are built on an internal application platform, ePages developers are highly dependent on third-party services.
+Read further, why it is not the easiest thing to integrate third-parties and how we tackle this.
+
+# The use case
+
+{% image blog-third-party-overview.png %}{% endimage %}
+
+Example: A customer buys something. They could choose e.g. PayPal as payment method and DHL as delivery method.
```

Comment:
"A customer buys something." --> "A customer buys something in an ePages online shop."